include-tips.html

```diff
@@ -1,7 +1,7 @@
 <h3>Tips:</h3>
     <ul>
         <li>Spell out acronyms with hyphens (e.g., AI becomes “aye-eye”).</li>
-        <li>If a word has an unusual pronunciation, consider rephrasing or substituting it with a synonym.</li>
+         <li>If a word has an unusual pronunciation, consider rephrasing or substituting it with a synonym.</li>
         <li>For pauses, use 2-3 commas (e.g., “Hello everyone ,,, my name is Howard”).</li>
         <li>Avoid empty slides (ensure every slide has speaker notes).</li>
         <li>Keep the browser open during video rendering.</li>
```

Comment:
I wonder if we can add a note about this where we ask for emails?

Reply:
Yeah good point.